app/jobs/deposit_status_job.rb

```diff
@@ -40,4 +44,12 @@ def parse_message(msg)
 
     raise "Unable to find required field 'druid' in payload:\n\t#{json}"
   end
+
+  def parent(object)
+    object.head.try(:collection) || object.head.work
```

Comment:
why is this necessary?  Isn't this just going from parent to child back to parent?

Reply:
That's a great question. If you set `event_context` on `object` directly, `object.head.deposit_complete!` does not get the `event_context`.

Reply:
Can you add a comment about that here? Something like, "although both objects represent the same row in the database, they are not the same object in memory"

Reply:
Comment added.